ipam_test.go

```diff
@@ -535,6 +535,80 @@ func TestSpace(t *testing.T) {
 	}
 }
 
+func TestConains(t *testing.T) {
+	tests := []struct {
+		Network          string
+		Subnet           string
+		ExpectedContains bool
+	}{
+		// Test 0.
+		{
+			Network:          "10.4.0.0/16",
```

Comment:
This is all magic to me so I may ask stupid questions. What if a network without CIDR is provided like `10.4.0.0`? Is there a requirement for the input schema? It would maybe be good to test that as well and maybe even use the actually used types defined by the interface we test.

Reply:
`ipam.Contains()` accepts `net.IPNet` as parameter. If test input parameter is invalid CIDR, `net.ParseCIDR()` in test body returns parse error and test fails. No need to test `net.ParseCIDR()` here specifically.